internal/framework/status/statuses.go

```diff
@@ -40,7 +40,7 @@ func (n *NginxGatewayStatus) APIGroup() string {
 }
 
 // ListenerStatuses holds the statuses of listeners where the key is the name of a listener in the Gateway resource.
```

Comment:
this is to reflect that is not a map anymore
```suggestion
// ListenerStatuses holds the statuses of listeners.
```